app/views/partials/_header.html.haml

```diff
@@ -14,6 +14,8 @@
                             - if current_user&.artist?
                                 = link_to 'New Campaign', new_campaign_path, remote: true, class: 'mui-btn mui-btn--flat mui-btn--primary'
                         %li
-                            = link_to 'Log in', new_user_session_path, remote: true, class: 'mui-btn mui-btn--flat mui-btn--primary'
+                            - unless current_user
+                                = link_to 'Log in', new_user_session_path, remote: true, class: 'mui-btn mui-btn--flat mui-btn--primary'
                         %li
-                            = link_to 'Sign up', new_user_registration_path, remote: true, class: 'mui-btn mui-btn--flat mui-btn--primary'
\ No newline at end of file
+                            - unless current_user
```

Comment:
Why two `if`statements and why not use `user_signed_in?` helper (more readable imo)

Reply:
Addressed in PR: https://github.com/CraftAcademy/venue/pull/19